test/paidComputeFlow.test.ts

```
@@ -1,219 +1,237 @@
+
+    const jsonMatch = output.match(
+      /Existing compute environments:\s*([\s\S]*)/,
+    );
+    if (!jsonMatch) {
+      console.error("Raw output:", output);
+      throw new Error("Could not find compute environments in the output");
+    }
+
+    let environments;
+    try {
+      environments = eval(jsonMatch[1]);
```

Biome (2.5.6)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)